RELEASE_NOTES.md

```diff
@@ -1,76 +1,46 @@
-# ChefDK 0.10.0 Release notes
+# ChefDK 0.11 Release notes
 
 ## New / Updated Packages
 
-* inspec - new dependency, version 0.9.0
-* kitchen-inspec - new dependency, version 0.9.0
-* knife-windows - updated to 1.1.1
-* chef-provisioning - updated to 1.5.0
-* chef-provisioning-aws - updated to 1.6.1
+This version of ChefDK includes a number of key updates to dependent packages:
 
-## Inspec
+* chef - updated to 12.7.2
+* ohai - updated to 8.10.0
+* inspec - updated to 0.11.0
+* kitchen-inspec - updated to 0.11.0
+* berkshelf - updated to 4.1.1
+* test-kitchen - updated to 1.5.0
+* knife-windows - updated to 1.2.1
+* foodcritic - new dependency, 6.0.0
+* rubocop - updated to 0.37.2
+* chef-provisioning - updated to 1.6.0
+* openssl - updated to 1.0.1r
+* CACerts - updated with latest root certificates
 
-[Chef Inspec](https://www.chef.io/inspec/) was released November 3rd.  We're now packaging this tool into the ChefDK.  The Github README has an [overview](https://github.com/chef/inspec#what-is-inspec) of the project and the developers posted an informative [blog](https://www.chef.io/blog/2015/11/04/the-road-to-inspec/) as well.
+## PolicyFile improvements and fixes
 
-The primary means of interfacing with Inspec is through its command line tool `inspec`.  The *most* important thing to note about the integration with the ChefDK right now is that the `inspec` tool is not added to your PATH by default when installing the ChefDK.  To leverage it you can either
+There are a number of PolicyFile updates and improvements:
 
-1. Prepend all your `inspec` invocations with `chef exec`.  For example, following their [guide](https://github.com/chef/inspec#usage) you would run `chef exec inspec exec test.rb` to run local Inspec tests.
-2. Add the executable to your PATH manually.  On linux/OSX systems this is located at `/opt/chefdk/embedded/bin` and on Windows it is located at `C:\opscode\chefdk\embedded\bin`.
+* Added support for community cookbooks
+* Improvements to native policy export
+* Improved policy logging
+* Better PolicyFile validation
 
-This path change is temporary - we plan to include `inspec` in the PATH by default in future ChefDK releases.
+### Breaking change
 
-## Kitchen-Inspec
+The ChefDK 0.11.0 policy files are not backward compatible with older versions of Chef Client.
+Once you upgrade to the ChefDK 0.11.0 policy files, you have to make sure you are running Chef Client 12.7 or later.
```

Comment:
This only applies to exported repos, but that's a big deal because test kitchen uses `chef export` under the hood. Maybe something like this:

> Repos created by the `chef export` command will only work with Chef Client 12.7 or later. The `policyfile_zero` provisioner for Test Kitchen uses `chef export` under the hood, so you will need to configure Test Kitchen to install Chef Client 12.7 or later.